### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by thiagodebastos** [jacogr/atom-gulp-control#2]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:55:11)
Object.activate (/Users/Thiago/.atom/packages/gulp-control/lib/gulp-control.coffee:10:9)
```

**Comment by thiago-om** [comment]
This happened when I tried to run gulp-watch, but everything worked fine running the default Gulp task, which includes watch.

**Comment by amitmtrn** [comment]
I fixed it at pull #4
